### PR TITLE
Add cards for Design Patterns and Feature Requests

### PR DESCRIPTION
**Pull request by huayunh**
I can no longer stand this...

#### Changes proposed in this Pull Request:
- Added two cards for design patterns and feature request

#### Screenshots:

image

### CODE REVIEW

**Comment by huayunh** [comment]
image

**Comment by joebochill** [comment]
If we're going to be in here making changes, I propose we:
- Drop the Theme Guidelines CTA — this is by far the weakest link we have here and it's on first position
- Rename Resources to Developer Resources (until we add a section for designers)

Maybe something like:
|          |          |          |
|------|------|------|
| Icons | Colors | Typography |
| Components | Design Patterns | Developer Resources |
| Github | Figma? | Feature Requests |

The new images also seem to stand out against the existing ones. I don't know if it's because they are more detailed (maybe we could simplify them to like a lightbulb with a thumbs up on it, and a basic shape with a ruler marker next to it?) or if it's because they are using muted background colors while the others are more saturated. Citron also happens to be my least favorite color 😆

**Comment by huayunh** [comment]
I tried to make them look more designed (rather than just simple shapes) :( 

Agreed on taking out the theme part.

**Comment by joebochill** [comment]
> I tried to make them look more designed (rather than just simple shapes) :(
> 
> Agreed on taking out the theme part.

I'm cool with a more designed style, but we may want to update some of the other ones too then.

**Comment by huayunh** [comment]
Made a story to continue the effort.

**Comment by joebochill** [comment]
I updated the CTAs. Also added a landing/home page for the style guide and the community page.

image
image
image

Looking for feedback on the graphics and the descriptions.

**Comment by huayunh** [comment]
Remember that when you add or remove new pages in `src/__configuration__/navigationMenu/navigation.tsx`, the file `doc-it/scripts/crawl/sitemap.json` will also need to be updated

**Comment by joebochill** [comment]
> Remember that when you add or remove new pages in `src/__configuration__/navigationMenu/navigation.tsx`, the file `doc-it/scripts/crawl/sitemap.json` will also need to be updated

Does this not happen automatically with `yarn indexer`?

**Comment by huayunh** [comment]
No. I didn't figure out how to make that happen since it involves exporting json file from a tsx file.

**Comment by joebochill** [comment]
image
image

File: src/docs/community/home.mdx [hunk 1]
```diff
@@ -1,13 +1,10 @@
+import { LinkGrid } from '../../app/components';
+import { links } from '../../__configuration__/community';
+
 <!-- KEYWORDS: bugs -->
```
Comment (huayunh) [inline src/docs/community/home.mdx:4]: Change this keyword too?

**Review by joebochill — APPROVED** [review]

**Pull request merged**